src/extension.ts

```diff
@@ -108,9 +108,7 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
     : defaultServerVersion;
   const serverProperties: string[] = workspace
     .getConfiguration("metals")
-    .get<string>("serverProperties")!
-    .split(" ")
-    .filter(e => e.length > 0);
```

Comment:
I've verified that the UI won't let you input a blank string (not even just spaces), so this I've removed this check.

Same thing below